### ISSUE TITLE
JS isn't being executed

### ISSUE THREAD

**Issue by perinikhil** [zalando/tailor#131]
I've tried to run the `basic-css-and-js` example and i've noticed that the JS isn't taking effect.
Also i get the following errors on the console:
![image](https://cloud.githubusercontent.com/assets/8386739/24292795/d1704066-10b4-11e7-9eea-86d9d8aafd69.png)

**Comment by smilingleo** [comment]
it seems something wrong with the `pipe.min.js`, when I replaced it with the content of `pipe.js`, it works.

**Comment by vigneshshanmugam** [comment]
Can you tell the steps to reproduce it? Did you do git clone <repo> and tried?

**Comment by perinikhil** [comment]
These were the steps I followed :)
On the terminal :-
```bash
$ git clone https://github.com/zalando/tailor.git
  # cloning done
$ cd tailor
$ npm install
  # install done

$ node examples/basic-css-and-js
Tailor server listening on port 8080
Fragment Server listening on port 8081
```
 On the browser :-
`http://localhost:8080/index`

**Comment by vigneshshanmugam** [comment]
Thanks guys, found the issue.. Please check the fix here #133
